backend/models/user.js

```diff
@@ -736,12 +736,14 @@ function _addProjects(account, username, models) {
 		query = { models: { $in: models } };
 	}
 
-	return Project.find({ account: account.account }, query).then(projects => {
+	return listProjects(account.account, query).then(projects => {
```

Comment:
Looking at the way _addProjects is called, `models` is always undefined. so we're just getting all the projects all the time.